Binary files dee-1.0.10/src/.dee-filter-model.h.swp and dee-1.0.10-r1/src/.dee-filter-model.h.swp differ
diff -urN dee-1.0.10/src/dee-filter.h dee-1.0.10-r1/src/dee-filter.h
--- dee-1.0.10/src/dee-filter.h	2012-02-25 01:46:45.000000000 +0900
+++ dee-1.0.10-r1/src/dee-filter.h	2012-04-26 00:49:44.645313412 +0900
@@ -30,7 +30,7 @@
 
 G_BEGIN_DECLS
 
-typedef struct _DeeFilter DeeFilter;
+// typedef struct _DeeFilter DeeFilter;
 
 /**
  * DeeFilterMapFunc:
diff -urN dee-1.0.10/src/dee-model-reader.h dee-1.0.10-r1/src/dee-model-reader.h
--- dee-1.0.10/src/dee-model-reader.h	2011-12-19 23:11:45.000000000 +0900
+++ dee-1.0.10-r1/src/dee-model-reader.h	2012-04-26 00:50:33.421317922 +0900
@@ -30,7 +30,7 @@
 
 G_BEGIN_DECLS
 
-typedef struct _DeeFilter DeeFilter;
+// typedef struct _DeeFilter DeeFilter;
 
 /**
  * DeeModelReaderFunc:
